src/core/sys/posix/sys/mman.d

```diff
@@ -98,45 +98,10 @@ PROT_EXEC
 PROT_NONE
 */
 
-version( linux )
-{
-    enum PROT_NONE      = 0x0;
-    enum PROT_READ      = 0x1;
-    enum PROT_WRITE     = 0x2;
-    enum PROT_EXEC      = 0x4;
-}
-else version( OSX )
-{
-    enum PROT_NONE      = 0x00;
-    enum PROT_READ      = 0x01;
-    enum PROT_WRITE     = 0x02;
-    enum PROT_EXEC      = 0x04;
-}
-else version( FreeBSD )
-{
-    enum PROT_NONE      = 0x00;
-    enum PROT_READ      = 0x01;
-    enum PROT_WRITE     = 0x02;
-    enum PROT_EXEC      = 0x04;
-}
-else version (Solaris)
-{
-    enum PROT_NONE = 0x00;
-    enum PROT_READ = 0x01;
-    enum PROT_WRITE = 0x02;
-    enum PROT_EXEC = 0x04;
-}
-else version (Android)
-{
-    enum PROT_NONE = 0x00;
-    enum PROT_READ = 0x01;
-    enum PROT_WRITE = 0x02;
-    enum PROT_EXEC = 0x04;
-}
-else
-{
-    static assert(false, "Unsupported platform");
-}
+enum PROT_NONE  = 0x0;
+enum PROT_READ  = 0x1;
+enum PROT_WRITE = 0x2;
+enum PROT_EXEC  = 0x4;
```

Comment:
[Must be defined in the POSIX standard, but values are not specified](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_mman.h.html).